config/config_params.go

```diff
@@ -124,6 +124,7 @@ type Config struct {
 	BPFKubeProxyIptablesCleanupEnabled bool           `config:"bool;true"`
 	BPFKubeProxyMinSyncPeriod          time.Duration  `config:"seconds;1"`
 	BPFKubeProxyEndpointSlicesEnabled  bool           `config:"bool;false"`
+	BPFToHostMark                      int            `config:"int;0"`
```

Comment:
Should try to make the name capture what this setting does once we pin that down `BPFExtToServiceConnmark` or something?

Reply:
Yup. Was originally meant to be a generic ToHostMark ... until it turned out it does not work ;-)